src/commands/start.rs

```diff
@@ -0,0 +1,60 @@
+use crate::api;
+use crate::models;
+use api::ApiClient;
+use chrono::Utc;
+use colored::Colorize;
+use models::ResultWithDefaultError;
+use models::TimeEntry;
+
+pub struct StartCommand;
+
+impl StartCommand {
+    pub async fn execute(
+        api_client: impl ApiClient,
+        billable: Option<bool>,
+        description: String,
+    ) -> ResultWithDefaultError<()> {
+        let running_entry_stop_time = Utc::now();
+        match api_client.get_running_time_entry().await {
+            Err(error) => {
+                println!(
+                    "{} {:?}",
+                    "Couldn't retrieve running time entry.".red(),
+                    error
+                );
+                return Ok(());
+            }
+            Ok(value) => {
+                if let Some(time_entry) = value {
+                    if let Err(stop_error) = api_client
+                        .update_time_entry(
+                            time_entry.as_stopped_time_entry(running_entry_stop_time),
+                        )
+                        .await
+                    {
+                        println!(
+                            "{} {:?}",
+                            "Couldn't stop running time entry.".red(),
+                            stop_error
+                        );
+                        return Ok(());
+                    }
+                    println!("Running time entry stopped")
```

Comment:
I think I would print the running entry that was stopped here, so the user has context of what entry was stopped.

also wdyt about making the "Running time entry stopped" a warning/yellow?

Reply:
oh yes, good ideas 👍

Reply:
Printing the stopped entry is probably nice as well